app/src/main/java/fr/free/nrw/commons/upload/UploadController.java

```diff
@@ -145,6 +145,7 @@ protected Contribution doInBackground(Void... voids /* stare into you */) {
                     } else {
                         contribution.setDateCreated(new Date());
                     }
+                    cursor.close();
```

Comment:
Ah, this line throws a NPE for me after the title/desc screen. I submitted #180 to try and fix it.